### ISSUE TITLE
Show results from eventbrite in search

### ISSUE THREAD

**Issue by niranjan94** [fossasia/open-event-orga-server#2676]
Prerequisites: #2674 #2675 

When a user searches for an event in eventbrite, the backend should also contact the scrapper and the query server and search for events in eventbrite and automatically import it into our site. (a link should be provided to the original site)

**Comment by satwikkansal** [comment]
Hi @niranjan94 , I'd like to work on this issue!

**Comment by mariobehling** [comment]
@satwikkansal Would be great. This is an important piece for the future of the project, but we will only incorporate it after the Beta-Release. Right now there are still a lot of bugs in the current version and we need to get the code base more stable. So, feel free to work on this, but understand that it will not be merged right away. Cheers.

**Comment by abhishek1995s** [comment]
Hi @niranjan94 I am cofounder of a startup working on a similar project . It is a mobile selection website . When user select a model name my website scarp all reviews, youtube comments, and twitter comments on behalf of the product perform machine learning and provide and overall picture to the user . I think i can make good contribution to this feature

**Comment by mariobehling** [comment]
As this is out of scope and we have a lot of tasks to get to the next version, I would say this is out of scope of the project for the time being. Closing it therefore.
